modules/troubleshoot-the-splunk-add-on.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_content-type: CONCEPT
+[id="troubleshoot-the-splunk-add-on_{context}"]
+= Troubleshoot the the Splunk add-on
+
+[role="_abstract"]
+If you stop receiving events from the {product-title} add-on, check the Splunk add-on debug logs for errors.
```

Comment:
```suggestion
If you stop receiving events from the {product-title} add-on, check the Splunk add-on debug logs for errors.
```

Reply:
I don't think I see a difference in this suggestion (unless I'm missing something)